src/webots/nodes/WbTrack.hpp

```diff
@@ -92,7 +92,7 @@ protected slots:
   // texture animation
   WbShape *mShape;
   WbTextureTransform *mTextureTransform;
-  WbVector2 mInitialTextureTransformTranslation;
+  QMap<QString, WbVector2> mInitialTextureTransformTranslations;
```

Comment:
A renaming of this member variable should be considered here as well (and maybe other member variables at other places).